backend/README.md

```diff
@@ -30,3 +30,9 @@ To fix VS Code import errors with Crow, try adding the following line to your `s
 ```
 
 [Source](https://stackoverflow.com/questions/58077908/linking-conan-include-to-vs-code)
+
+## Testing
+After building, you can run tests from `/backend`:
+```bash
+cd build && ./tests
```

Comment:
**thought**: I definitely think it's better to have `make check`. I'll get on that.